plugins/command-line/prism-command-line.js

```diff
@@ -4,6 +4,67 @@ if (typeof self === 'undefined' || !self.Prism || !self.document) {
 	return;
 }
 
+// Scoped to this IIFE so they can be accessed by both hooks. -- cwells
+var outputLines = [];
```

Comment:
At the moment plugins and languages use the `env` object directly for their variables. For example `php` creates the property `env.backupCode` and the `keep-markup` plugin uses `env.keepMarkup`. As you rightly point out this practice could lead to conflicts. I like your proposed solution, but maybe we should use a shorter name like:

```js
env.vars['command-line'].outputLines = [];
```